otter_welcome_buddy/database/db_helpers.py

```diff
@@ -0,0 +1,172 @@
+from collections import namedtuple
+from collections.abc import Callable
+from pathlib import Path
+from sqlite3 import Cursor
+from typing import Any
+
+from otter_welcome_buddy.common.constants import DATA_FILE_PATH
+from otter_welcome_buddy.database.dbconn import DbContext
+
+
+def namedtuple_factory(cursor: Cursor, row: tuple) -> tuple:
+    """Returns sqlite rows as named tuples"""
+    fields: list[str] = [col[0] for col in cursor.description if col[0].isidentifier()]
+    named_row = namedtuple("Row", fields)  # type: ignore
+    return named_row(*row)
+
+
+def dict_factory(cursor: Cursor, row: tuple) -> dict:
+    """Returns sqlite rows as dict"""
+    col_names = [col[0] for col in cursor.description]
+    return dict(zip(col_names, row))
+
+
+def insert_one(table: str, columns: list[str], values: tuple) -> int:
```

Comment:
Insert_<entity>_into it's more clear, what do you think?

Reply:
Don't think so, by default insert is into and this method is specific for just one value.